app/utils/CheckYourAnswersHelper.scala

```diff
@@ -30,6 +30,18 @@ import viewmodels.AnswerRow
 
 class CheckYourAnswersHelper @Inject()(countryOptions: CountryOptions)(userAnswers: UserAnswers)(implicit messages: Messages) {
 
+  def trusteesNino(index: Int): Option[AnswerRow] = userAnswers.get(TrusteesNinoPage(index)) map {
+    x =>
+      AnswerRow(
+        "trusteesNino.checkYourAnswersLabel",
+        HtmlFormat.escape(x),
+        routes.TrusteesNinoController.onPageLoad(CheckMode, index).url,
+        getTrusteeFirstLastName(index)
+      )
+  }
+
+  def getTrusteeFirstLastName(index: Int): String = userAnswers.get(TrusteesNamePage(index)).get.toString
```

Comment:
Please remove this method as it duplicates what is in the companion object.

Reply:
Done